### PR TITLE
Task general

### PR DESCRIPTION
**Pull request by grace-p-03**
Please ignore the fact that I haven't made a new branch. doh! The bit i would like to be reviewed is my 'about' section.

### CODE REVIEW

File: normalize.css [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+body {
+  margin: 0;
+}
+
+/**
+ * Render the `main` element consistently in IE.
+ */
+
+main {
+  display: block;
```
Comment (gabrielrowan) [inline normalize.css:32]: Do you need display block here? As block is the default

File: portfolio.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	<link href="style.css" rel="stylesheet" type="text/css" />
+	<meta name="viewport" content="width=device-width">
+  </head>
+    <body>
+	  <nav>	
+	  <div class="logo">Grace Palacz</div>
+	    <div class="navLinks">
+		  <a href="#about">About</a>
+		  <a href="#portfolio">Portfolio</a>
+	      <a href="#contact">Contact</a>
```
Comment (gabrielrowan) [inline portfolio.html:15]: This could be indented in line with the other  links

File: portfolio.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+		  <a href="#portfolio">Portfolio</a>
+	      <a href="#contact">Contact</a>	
+	    </div>
+	  </nav> 
+	    <header>
+		  <div class="intro">
+	        <h1>Hello! I'm Grace</h1>
+			  <p>I'm a full stack developer based in the South West.</p>				  
+		  </div>	  
+		</header>
```
Comment (gabrielrowan) [inline portfolio.html:23]: The closing and opening header tag have different indentation

File: style.css [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+}
+
+body {
+	font-family: walsheim;
+}
+
+nav {
+	position: fixed;
+	width: 100%;
+	top: 0;
```
Comment (gabrielrowan) [inline style.css]: Is there a reason for the top: 0 and left: 0 here? What do you need to achieve by including them?

File: portfolio.html [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+	  </nav> 
+	    <header>
+		  <div class="intro">
+	        <h1>Hello! I'm Grace</h1>
+			  <p>I'm a full stack developer based in the South West.</p>				  
+		  </div>	  
+		</header>	
+		  <section class="about">
+		    <div class="profileImage">
+		      <h2>About me</h2>
```
Comment (gabrielrowan) [inline portfolio.html:26]: Do your h2 and img tag need the same indentation?

File: portfolio.html [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+			    <img src="1516887791047.jpeg" alt="An image of Grace Palacz"> 
+		    </div>
+			  <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam facilisis risus neque, non pretium dui pretium eu.                       Aenean lectus urna, luctus eu ante id, lobortis tincidunt libero. Cras at accumsan ante, vel semper mi. Praesent vel                      arcu at lectus iaculis egestas. Vivamus tincidunt erat eu nulla tincidunt mollis nec eu lectus. Vestibulum euismod enim                   non urna pharetra, et elementum lacus pellentesque. In interdum elit id felis finibus vehicula. Morbi mattis nisl                         varius, tempus ipsum vel, faucibus nisi. Mauris eget magna vel nunc fermentum pellentesque.</p> 
+		  </section>
+			<div class="projectsIntro">
```
Comment (gabrielrowan) [inline portfolio.html:32]: Do your opening and closing divs need the same indentation?

**Review by gabrielrowan — CHANGES_REQUESTED** [review]
There wasn't a lot I could find to comment on here - basically just indentation and a line that I'm not sure is necessary 

I think the indentation is optional to change as it doesn't affect how the website works

**Review by ethan-garrett — APPROVED** [review]
Looks good to me

**Pull request merged**